recipes/chemdataextractor/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "ChemDataExtractor" %}
+{% set version = "1.3.0" %}
+{% set sha256 = "2c989cfb60048dc57a49472228381574bff351c31896bdcd621991f68c0f9cc9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  entry_points:
+    - cde = chemdataextractor.cli:cli
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - appdirs
+    - beautifulsoup4
+    - click
+    - cssselect
+    - dawg
+    - lxml
+    - nltk
+    - pdfminer.six
+    - python-crfsuite
+    - python-dateutil
+    - pyyaml
+    - requests
+    - six
+
+test:
+  commands:
+    - cde
+  imports:
+    - chemdataextractor
```

Comment:
It may be worthwhile to test the imports of the submodules as well just to potentially catch any missing dependencies (not that I think there necessarily are any).